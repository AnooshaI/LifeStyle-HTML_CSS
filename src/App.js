import './App.scss';
import React from 'react'
import { Body } from '../src/components/body/body.jsx'
import { Header } from '../src/components/header/header.jsx'

function App() {
  return (
    <div className="App">
     <Header/>
     <Body/>
    </div>
  );
}

export default App;
