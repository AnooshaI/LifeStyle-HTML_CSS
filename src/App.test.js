import React from "react";
import Enzyme, { shallow, mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import App from "./App";
import renderer from "react-test-renderer";
Enzyme.configure({
  adapter: new Adapter(),
});
//Setup function is to create wrapper for your mount so you dont have to define it again
const setUp = () => {
  const wrapper = mount(<App />); //change componant name here
  return wrapper;
};
describe("App", () => {
  it("should render correctly", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
