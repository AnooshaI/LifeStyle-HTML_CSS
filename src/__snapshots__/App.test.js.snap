// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div
  className="App"
>
  <div
    className="main"
  >
    <div
      className="header"
    >
      <div
        className="logo"
      >
         
        <img
          alt="Logo"
          src="test-file-stub"
        />
      </div>
      <div
        className="nav"
      >
        <nav>
          <div
            className="navItems"
          >
            <a
              href="#"
            >
              World   
            </a>
            <a
              href="#"
            >
               Sports   
            </a>
            <a
              href="#"
            >
               Opinion
            </a>
            <a
              href="#"
            >
               Lifestyle
            </a>
            <a
              href="#"
            >
               Business
            </a>
            <a
              href="#"
            >
               Culture
            </a>
            <a
              href="#"
            >
               Fashion
            </a>
            <a
              href="#"
            >
               Tech
            </a>
          </div>
        </nav>
      </div>
      <div>
          
        <img
          alt="DP"
          className="Dp"
          src="test-file-stub"
        />
         
      </div>
    </div>
  </div>
  <div>
    <div
      className="Page"
    >
      <div
        className="homePage"
      >
        <div
          className="content"
        >
          <h1
            className="title"
          >
            Lifestyle.
            <br />
          </h1>
          <p
            className="subtitle"
          >
            The Latest and best lifestyle article selected by our editorial office.
          </p>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="column"
        >
          <div
            class="cards"
            id="card1"
          />
        </div>
        <div
          className="column"
        >
          <div
            class="cards"
            id="card2"
          />
        </div>
        <div
          className="column"
        >
          <div
            class="cards"
            id="card3"
          />
        </div>
        <div
          className="column"
        >
          <div
            class="cards"
            id="card4"
          />
        </div>
        <div
          className="column"
        >
          <div
            class="cards"
            id="card5"
          />
        </div>
        <div
          className="column"
        >
          <div
            class="cards"
            id="card6"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
